ui/app/AppLayouts/Chat/stores/CommunitiesStore.qml

```diff
@@ -11,7 +11,7 @@ QtObject {
                              infiniteSupply, transferable, selfDestruct, chainId, artworkSource)
     {
         communityTokensModuleInst.deployCollectible(communityId, address, name, symbol, description, supply,
-                                                    infiniteSupply, transferable, selfDestruct, chainId, artworkSource)
+                                                    infiniteSupply, transferable, selfDestruct, chainId, String(artworkSource).replace("file://", ""))
```

Comment:
It's better to use `formatImagePath` instead, as it makes it work on Windows as well. You can use it from the backend too so we don't do a back and forth between the backend and the frontend.

Reply:
@jrainville done